src/Features/Core/Portable/SolutionCrawler/AggregateIncrementalAnalyzer.cs

```diff
@@ -61,6 +61,14 @@ public async Task DocumentCloseAsync(Document document, CancellationToken cancel
             }
         }
 
+        public async Task ActiveDocumentSwitchedAsync(TextDocument document, CancellationToken cancellationToken)
```

Comment:
💡 There are a bunch of uses of the term 'switched'. These should all be updated to 'changed' for consistency with existing code.

Reply:
Details added in https://github.com/dotnet/roslyn/pull/57172#discussion_r803314568, this was an intentional naming change to avoid confusion.